syntax = "proto3";

package request;

message WriteRequest {
    string filepath = 1;
    string data = 2;
    int32 offset = 3;
}